samples: 802154_sniffer: add `bootloader` command. (#20879)
nrfconnect/sdk-nrf: samples/peripheral/802154_sniffer/README.rst

      sleep

bootloader - reboot the device to the bootloader
================================================

> The ``bootloader`` command reboots the device without skipping the bootloader.

Reviewer: ```suggestion
The ``bootloader`` command reboots the device into the bootloader mode.
```

   .. parsed-literal::
      :class: highlight

>       sleep

Reviewer: Should be `bootloader` here. Also if the bootloader prints something or blinks some LED, it would be good to mention that or link to some documentation page.

Author: right. Added a line about red LED pulsing after jumping to bootloader

      sleep

bootloader - reboot the device to the bootloader
================================================

> The ``bootloader`` command reboots the device into the bootloader mode.

Reviewer 2: ```suggestion
The ``bootloader`` command reboots the device in bootloader mode.
```

   .. parsed-literal::
      :class: highlight

      bootloader

> The device will reboot into the bootloader mode, and the red LED will start pulsing.

Reviewer 2: ```suggestion
The device reboots into bootloader mode, and the red LED starts pulsing.
```
